inlong-sort/sort-flink/base/src/main/java/org/apache/inlong/sort/base/schema/SchemaChangeHelper.java

```diff
@@ -0,0 +1,306 @@
+                }
+                LOGGER.warn("Extract schema change type failed, origin schema: {}", originSchema, e);
+            }
+            if (types == null) {
+                continue;
+            }
+            if (types.size() == 1) {
+                SchemaChangeType type = types.stream().findFirst().get();
+                typeMap.computeIfAbsent(type, k -> new ArrayList<>()).add(alterColumn);
+            } else {
+                // Handle change column, it only exists change column type and rename column in this scenario for now.
+                for (SchemaChangeType type : types) {
+                    SchemaChangePolicy policy = policyMap.get(type);
+                    if (policy == SchemaChangePolicy.ENABLE) {
+                        LOGGER.warn("Unsupported for {}: {}", type, originSchema);
+                    } else {
+                        doSchemaChangeBase(type, policy, originSchema);
+                    }
+                }
+            }
+        }
+
+        if (!typeMap.isEmpty()) {
+            doAlterOperation(database, table, originData, originSchema, data, typeMap);
+        }
+    }
+
+    @Override
+    public String doAddColumn(SchemaChangeType type, String originSchema) {
+        SchemaChangePolicy policy = policyMap.get(SchemaChangeType.DROP_TABLE);
+        if (policy == SchemaChangePolicy.ENABLE) {
+            throw new SchemaChangeHandleException(String.format("Unsupported for %s: %s", type, originSchema));
+        }
+        doSchemaChangeBase(type, policy, originSchema);
+        return null;
+    }
+
+    @Override
+    public String doChangeColumnType(SchemaChangeType type, String originSchema) {
+        SchemaChangePolicy policy = policyMap.get(SchemaChangeType.DROP_TABLE);
+        if (policy == SchemaChangePolicy.ENABLE) {
+            throw new SchemaChangeHandleException(String.format("Unsupported for %s: %s", type, originSchema));
+        }
+        doSchemaChangeBase(type, policy, originSchema);
+        return null;
+    }
+
+    @Override
+    public String doRenameColumn(SchemaChangeType type, String originSchema) {
+        SchemaChangePolicy policy = policyMap.get(SchemaChangeType.DROP_TABLE);
+        if (policy == SchemaChangePolicy.ENABLE) {
+            throw new SchemaChangeHandleException(String.format("Unsupported for %s: %s", type, originSchema));
+        }
+        doSchemaChangeBase(type, policy, originSchema);
+        return null;
+    }
+
+    @Override
+    public String doDropColumn(SchemaChangeType type, String originSchema) {
+        SchemaChangePolicy policy = policyMap.get(SchemaChangeType.DROP_TABLE);
+        if (policy == SchemaChangePolicy.ENABLE) {
+            throw new SchemaChangeHandleException(String.format("Unsupported for %s: %s", type, originSchema));
+        }
+        doSchemaChangeBase(type, policy, originSchema);
+        return null;
+    }
```

Comment:
return null？ Why not to set method return type to void.

Reply:
Doris connector may use the return value of this method

Reply:
Fixed